IanGRap/Squish
# Bugfix: Issue #74 (Use Prefab In Block Formations)

Contributors: Austin

Goal: Fix the bug where the block formations do not use up-to-date versions of the block prefab

What Changed:
- In Spawner.cs, instead of instantiating a block formation object, use the positions of the blocks within the prefab to instantiate block prefab objects.

How to Test
- Load up the 2017 scene and make sure blocks spawn
- Test in standalone build as well
- Change something in the block prefab and ensure that the changes get applied in spawned blocks
